### ISSUE TITLE
Refine code as jQuery plugin to use it's sizzle selector and it's animation as css3 fallback

### ISSUE THREAD

**Issue by salisa** [usablica/motion.js#15]
#### Greetings:

> Congratulation Afshin,
> The motion.js is pretty cool in vision of iranin javaScript developers.
> [_Salisa_](http://github.com/salisa)

As a good jQuery based css3 transions  projects like [jQuery Transit](http://ricostacruz.com/jquery.transit) by Rico Sta. Cruz developement on this field comes into a challenge!

The successful motion.js may use jQuery's power because of:
1. The number one css selector (sizzle engine)
2. Terrific animation functions (use as fallback css3 transition in many cases)
3. Decrease the size of motion.js source code in next development generations

Good Luck

**Comment by afshinm** [comment]
Hey @salisa,

First of all, thanks for your greeting.
I have to mention that the vision of MotionJs is to provide a better way for creating CSS3 animations, only. So we only focus of CSS3 animations.

Really thanks for your suggestion, there are very cool and interesting but I should say:
1. We are using native JavaScript CSS selector engine. It's fast, no dependency.
2. We need no fallback on animations, only CSS3 supported.
3. Sure.

Thanks again, let me know what you think.

**Comment by salisa** [comment]
Well, It's the fancy roadmap, Great!
As another issue:
The alert informing for old browser users is not such a good pleasure of interaction!
you can make it configable with a silent mode,
or throwing manual exception by default:

``` javascript
throw new Error("Your browser is out of date, use modern one!");
```

**Comment by afshinm** [comment]
Yeah, I agree with you about that. We'll fix that in next version but it would be so better if you become a MotionJs contributor :)
